### PR TITLE
improve grammatical correctness of alternatives

### PR DESCRIPTION
**Pull request by lsmith77**
adds https://github.com/gambolputty/german-nouns and https://github.com/tomaarsen/Inflex

### CODE REVIEW

File: tests/test_grammatically_correct_alternatives/test_german_plural/output.json [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                    "text": "schwieriges Themen"
+                },
+                {
+                    "text": "komplexe Materien"
+                },
+                {
+                    "text": "Fragestellungen"
+                },
+                {
+                    "text": "Schwachpunkt"
```
Comment (lsmith77) [inline tests/test_grammatically_correct_alternatives/test_german_plural/output.json]: https://github.com/gambolputty/german-nouns/issues/8

File: tests/test_grammatically_correct_alternatives/test_german_plural/output.json [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+                },
+                {
+                    "text": "Beeinträchtigungen"
+                },
+                {
+                    "text": "Erschwernis"
```
Comment (lsmith77) [inline tests/test_grammatically_correct_alternatives/test_german_plural/output.json]: https://github.com/gambolputty/german-nouns/issues/8

File: tests/test_grammatically_correct_alternatives/test_german_plural/output.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+                    "text": "Fehler"
+                },
+                {
+                    "text": "Bürden"
+                },
+                {
+                    "text": "Hürden"
+                },
+                {
+                    "text": "Fallstricke"
```
Comment (lsmith77) [inline tests/test_grammatically_correct_alternatives/test_german_plural/output.json:32]: https://github.com/gambolputty/german-nouns/issues/8

**Comment by lsmith77** [comment]
@lnazarenko I managed to get it to work this way .. but maybe it would make sense to adopt https://github.com/bjascob/LemmInflect

**Review by lnazarenko — APPROVED** [review]

**Pull request merged**